code/script.js

```diff
@@ -1,9 +1,8 @@
-// A function that adds and remove the class "active" on the section you click on.
-// We haven't really talked about ´this´ yet, but we will... ;)
-// console.log(this) to see how it works
-function toggle() {
-  this.classList.toggle("active")
-}
+const accordion = document.querySelectorAll('#button') 
 
-// Selects an HTML element, and calls a function which will be executed when the element is clicked.
-document.getElementById("section1").onclick = toggle
+accordion.forEach((accordion) => {
+  accordion.addEventListener('click', () => {
+    accordion.classList.toggle('open')
+  })
+ })
```

Comment:
In order to not get confused with what is what, try this:
- keep your function named `accordion` 
- rename the parameter i.e. what is within the function's parenthesis `(accordion)` to something more descriptive like `accordionButton`

Example below. Remember that you can write literally anything as the parameter to describe what it is. So what you're code is saying is in the accordion, for each accordion button, do something when the button is clicked.
 ```javascript
accordion.forEach((accordionButton) => {
  accordionButton.addEventListener('click', () => {
    accordionButton.classList.toggle('open')
  })
 })